descriptors/ModuleDescriptor-template.json

```diff
@@ -220,6 +220,19 @@
             "circulation-storage.loans.item.delete"
           ]
         },
+        {
```

Comment:
I think we have to update API version as well.

Reply:
Sure. I'm moving this PR to another branch and update it.